### PR TITLE
[Number4] Created Profile form in Dashboard page

### PR DESCRIPTION
**Pull request by KoutarouTakayama**
_**Commands to run**_
-php artisan storage:link
-php artisan migrate:fresh

_**Preconditions**_
-`/dashboard` is displayed.
-In this PR, Users can't upload images so it is needed that image file is uploaded in `storage/app/public` directory and the file name has to exist in the user's `avatar_url` column of users table for checking this function.

_**Expected outputs**_
-Profile card is displayed.
-User's name is displayed on the profile.
-User's avatar is displayed on the profile.
-Learned words number and its link is displayed on the profile **but the link doesn't work yet.**
-Learned lessons number and its link is displayed on the profile **but the link doesn't work yet.**

_**memo**_
-I changed some variable and functions' name for making to easier understand.
-I added `word` column in words table.
-I didn't change anything in app/Http/Middleware/RedirectIfAuthenticated.php so please just ignore.
-The reason why I deleted seeder and factory is I changed them for easier making data but I thought it has no relationship with this project.

_**asana URL**_
-https://app.asana.com/0/1107951271303646/1108346470095437

_**Wireframe**_
https://docs.google.com/presentation/d/1a0mVYW0HPXVJn5pCqQp9an6AA6Kuq515HaVuDW9b030/edit#slide=id.g3cc2945e97_0_11

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -2,12 +2,18 @@
...
+use App\User;
 use Illuminate\Http\Request;
 
 class UserController extends Controller
 {
     public function dashboardView()
     {
-        return view('/user/dashboard');
+        $learnedWordsNumber = app(User::class)::find(auth()->id())->learnedWord->count();
+        $learnedLessonsNumber = app(User::class)::find(auth()->id())->learnedLesson->count();
```
Comment (jeremiahC) [inline app/Http/Controllers/UserController.php]:
```
  public function dashboardView()
    {
        $learnedWordsNumber = Auth::user()->learnedWord->count();
        $learnedLessonsNumber = Auth::user()->learnedLesson->count();
        ... some code
```

so you don't need to use `find()` to get the authenticated user, just call that directly an you have an authenticated user including getting it's id.

**Comment by KoutarouTakayama** [comment]
I don't know which expression is better but I changed it into `auth()->user()->` not `Auth::user()->`.
This `auth()->user()->` expression is more natural for me but if it's not good, please tell me.

**Comment by jeremiahC** [comment]
there is a difference in accessing the object, yours accesses the user function by function, the `Auth::user()` accesses instantly, but they perform the same behaviour, I suggest to use the `Auth::user()`.

**Comment by KoutarouTakayama** [comment]
Thank you for advising me. I changed them.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**